packages/protoplugin/src/safe-identifier.ts

```diff
@@ -83,7 +83,11 @@ const reservedIdentifiers = new Set([
 ]);
 
 /**
- * Escapes names that are reserved identifiers in ECMAScript, TypeScript.
+ * Escapes reserved words in ECMAScript and TypeScript identifiers, by appending
+ * a dollar sign.
+ *
+ * This function is intended for use with identifiers from Protobuf. The passed
+ * string must be a valid identifier (e.g. not start with a digit).
```

Comment:
Not directly related, but I noticed the docs for the function `safeIdentifier` need clarification.